README.md

```diff
@@ -17,32 +17,34 @@ The following steps should only be used to install if you are planning on contri
 1. Install [Node.js 16.x](https://nodejs.org/en/download/)
 1. Install [Rust](https://www.rust-lang.org/learn/get-started).
 1. Install [Yarn](https://classic.yarnpkg.com/en/docs/install).
+1. Run `yarn install` from the root directory to install packages (see sections below for OS specific details).
 1. Windows:
    1. Install [Build Tools for Visual Studio 2019](https://docs.microsoft.com/en-us/visualstudio/releases/2019/history#release-dates-and-build-numbers), including the C++ Build Tools and the Windows 10 SDK, for Rust.
    1. Rust builds also require installing [MinGW-w64 via win-builds](http://win-builds.org/doku.php/download_and_installation_from_windows).
       1. Choose `Native Windows`, `x86_64`, choose an empty directory, and click OK.
       1. On the next screen, click `Process`.
       1. Once it finishes, add the `bin` folder containing `cc` to your PATH environment variable.
-1. Run `yarn install` from the root directory to install packages.
-
+1. MacOS:
    - If `yarn install` fails with an error that includes "Failed to build cmake", you may need to first install cmake. For example, on macOS:
 
      1. Run `brew install cmake`, you'll need cmake version 3.15 or higher.
 
    - If `yarn install` fails with an error that includes "Could NOT find OpenSSL", you may need to first install OpenSSL and add an environment variable. For example, on macOS:
 
      1. Run `brew install openssl`
-     1. Run `` export OPENSSL_ROOT_DIR=`brew --prefix openssl`  ``
+     1. Run `brew link --force openssl`
```

Comment:
I wanted to talk about this change.

`OPENSSL_ROOT_DIR` before is read by the [node-datachannel](https://www.npmjs.com/package/node-datachannel) build scripts only, so were installing openssl in brew, and manually referencing it from brew.

By using link --force, this is including the openssl libs in your default $PATH so it'll get picked up by everything as the default openssl instead of the one included with mac.

In the [documentation](https://docs.brew.sh/How-to-Build-Software-Outside-Homebrew-with-Homebrew-keg-only-Dependencies) for link and kegs it says this,

```
A number of people in this situation are either forcefully linking keg-only tools with brew link --force or moving default system utilities out of the PATH and replacing them with manually created symlinks to the Homebrew-provided tool.


Please do not remove macOS native tools and forcefully replace them with symlinks back to the Homebrew-provided tool. Doing so can and likely will cause significant breakage when attempting to build software.```

What do you think?

Reply:
Yea I think you are right. It might not actually be a problem for OpenSSL since LibreSSL (I think this might be a backward compatibility thing, but not sure I want to go down that rabbit hole), probably best to heir on the side of caution.

Also if this is only used for the build scripts of a dependency, then we shouldn't need it more than in the current install session. My only concern is that the user wipes the `node_modules` and then tries to reinstall later, the install script would break because the export is temporary.

Either way happy to remove.